code/game/objects/items/devices/chameleonproj.dm

```diff
@@ -209,15 +209,15 @@
 	if(active)
 		to_chat(user, "<span class='notice'>You reconfigure [src].</span>")
 		activate(user)
+		return
+	to_chat(user, "<span class='notice'>You activate [src].</span>")
+	apply_wibbly_filters(user)
+	if(do_after(user, 50, target = user) && user.cell.use(activationCost))
```

Comment:
```suggestion
	if(do_after(user, 5 SECONDS, target = user) && user.cell.use(activationCost))
```

Reply:
I did that?
I think git is shitting the bed currently

Reply:
That was fixed there when I reviewed it, wtf how is it reverted

Reply:
Git imploded but it's there now